### PR TITLE
add autochess vng api

### PR DESCRIPTION
**Pull request by didadadida93**
- [x] My submission is formatted according to the guidelines in the [contributing guide](CONTRIBUTING.md)
- [x] My addition is ordered alphabetically
- [x] My submission has a useful description
- [x] The description does not end with punctuation
- [x] Each table column is padded with one space on either side
- [x] I have searched the repository for any relevant issues or pull requests
- [x] Any category I am creating has the minimum requirement of 3 items
-  @[x] All changes have been [squashed][squash-link] into a single commit

[squash-link]: <https://github.com/todotxt/todo.txt-android/wiki/Squash-All-Commits-Related-to-a-Single-Issue-into-a-Single-Commit>

### CODE REVIEW

**Comment by marekdano** [comment]
ref #1620

File: README.md [hunk 1]
```diff
@@ -425,6 +425,7 @@ API | Description | Auth | HTTPS | CORS |
 |---|---|---|---|---|
 | [Age of Empires II](https://age-of-empires-2-api.herokuapp.com) | Get information about Age of Empires II resources | No | Yes | No |
 | [AmiiboAPI](https://amiiboapi.com/) | Nintendo Amiibo Information | No | Yes | Yes |
+| [Autochess VNG API](https://github.com/didadadida93/autochess-vng-api) | Rest Api for Autochess VNG | No | Yes | Yes |
```
Comment (marekdano) [inline README.md:428]: The validation format check felt here: 
Screenshot 2021-04-29 at 15 30 29

Can you please remove `API` from the title?

**Review by marekdano — CHANGES_REQUESTED** [review]
Thanks for opening the PR @didadadida93!
Can you please update your record?

**Comment by didadadida93** [comment]
> Thanks for opening the PR @didadadida93!
> Can you please update your record?

done :+1:

**Review by marekdano — APPROVED** [review]

**Pull request merged**